fix!: Remove `url:` import feature (#2456)
wxt-dev/wxt: docs/guide/resources/migrate.md

3. Move public `assets/*` into the `public/` directory
4. If you use CSUI, migrate to WXT's `createContentScriptUi`
5. Convert Plasmo's custom import resolutions to Vite's

> 6. Importing remote code via a URL is not supported in WXT anymore, see [issue](https://github.com/wxt-dev/wxt/issues/2262). Download the file yourself and host it in the repo yourself.

Reviewer: ```suggestion
6. Importing remote code via a URL is not supported in WXT, see [this issue](https://github.com/wxt-dev/wxt/issues/2262) for alternatives
```
